README.md

```diff
@@ -189,6 +189,13 @@ vscode.commands.registerCommand(
 );
 ```
 
+## Formatting
+
+When `rubyLsp.formatter` is set to `auto`, Ruby LSP tries to determine which formatter to use.
+
+If the bundle has a **direct** dependency on a supported formatter, such as `rubocop` or `syntax_tree`, that will be used.
```

Comment:
(this will apply only after https://github.com/Shopify/ruby-lsp/pull/654 is merged).